sources/packages/web/src/views/student/StudentDashboard.vue

```diff
@@ -1,33 +1,37 @@
 <template>
   <student-page-container layout-template="centered">
-    <formio
-      formName="studentwelcomepage"
+    <formio-container
+      formName="studentWelcomePage"
+      :formData="formData"
       @customEvent="goToStudentApplication"
-    ></formio>
+    />
   </student-page-container>
 </template>
 <script lang="ts">
-import { StudentRoutesConst } from "../../constants/routes/RouteConstants";
-import { computed } from "vue";
-import { useStore } from "vuex";
+import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
 import { useRouter } from "vue-router";
+import { onMounted, reactive } from "vue";
+import { StudentService } from "@/services/StudentService";
 
 export default {
   setup() {
-    const store = useStore();
     const router = useRouter();
-    const user = computed(() => store.state.student.profile);
-
+    const formData = reactive({ firstName: "" });
     const goToStudentApplication = async () => {
       await router.push({
         name: StudentRoutesConst.STUDENT_APPLICATION_FORM,
       });
     };
 
+    onMounted(async () => {
```

Comment:
it is possible to get the first name from the student store in profile date? so that we student moves back and forth to the dashboard API call can be avoided.

Reply:
student store doesn't have that info. as per the discussion with signori, I am passing the whole API data to form. so that the designer will get access to the other student variables

Reply:
What is the benefit of using API response instead of state? Is the student dashboard expected to display any data which is not from student profile?

Reply:
The data is not persisted in the store because it was not consumed anywhere. That said, it can be added to the store since the API call is already happening to get SIN status.

Reply:
@ann-aot, to be clear, our conversation was about a recommendation to add the entire payload instead of only the name.
If the "payload" is the student store data or the API I do not see a big difference.